packages/grafana-plugin-configs/webpack.config.ts

```diff
@@ -179,18 +183,6 @@ const config = async (env: Record<string, unknown>): Promise<Configuration> => {
             },
           ],
         },
-        {
-          dir: path.resolve(DIST_DIR),
-          files: ['package.json'],
-          rules: [
-            {
-              search: `"version": "${getPackageJson().version}"`,
-              replace: env.commit
-                ? `"version": "${getPackageJson().version}-${env.commit}"`
-                : `"version": "${getPackageJson().version}"`,
```

Comment:
Does this impact the gh workflow that releases an external plugin in someway? 🤔

Reply:
I checked and it shouldn't: it gets the version from the plugin.json and for the rest, it should be like any other external plugin release.

Reply:
Fair enough. Not sure how this made its way here then. 🤷